src/main/java/org/fundacionjala/at15/pokemon/Town.java

```diff
@@ -7,32 +7,48 @@
     private ArrayList<Trainer> trainers;
     private Store store;
     private static final int POKEMON_HP = 100;
-    // Center center;
-    // Gym gym;
+    private Center center;
+    private Gym gym;
 
     public Town() {
         wildPokemons = new ArrayList<Pokemon>();
         trainers = new ArrayList<Trainer>();
         store = new Store();
-        // center = new Center();
-        // gym = new Gym();
+        center = new Center();
+        gym = new Gym();
     }
 
     public void addWildPokemonsToTown(int wildPokemonsTotal) {
         for (int ind = 0; ind < wildPokemonsTotal; ind++) {
-            wildPokemons.add(new Pokemon(POKEMON_HP, "Pokemon" + ind));
+            wildPokemons.add(ind, new Pokemon(POKEMON_HP, "Pokemon"));
         }
     }
 
     public void addTrainersToTown(int trainersTotal) {
         for (int ind = 0; ind < trainersTotal; ind++) {
             Pokemon pokemon = new Pokemon(POKEMON_HP, "Pikachu");
-            trainers.add(new Trainer(pokemon, "Trainer" + ind));
+            trainers.add(ind, new Trainer(pokemon, "Trainer"));
         }
     }
 
+    public Center getCenter() {
+        return center;
+    }
+
     public Store getStore() {
         return store;
     }
 
+    public Gym getGym() {
+        return gym;
+    }
+
+    public ArrayList<Pokemon> getPokemonsOfTheTown() {
```

Comment:
to be consistent with the attribute's name, this method could be getWildPokemons()

Reply:
Thanks Adhemar we will change the names to be more consistents